merlinx.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "acex88",
+			"oem": "Redmi",
+			"device": "Redmi Note 9",
+			"filename": "crDroidAndroid-14.0-20231229-merlinx-v10.0.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/merlinx/10.x/crDroidAndroid-14.0-20231229-merlinx-v10.0.zip/download",
```

Comment:
upload build

Reply:
Done

Reply:
not proper file
you uploaded **crDroidAndroid-14.0-20231230-merlinx-v10.0.zip**
either update json from that build, or upload proper file